back/engines/free/email_campaigns/app/models/concerns/email_campaigns/schedulable.rb

```diff
@@ -11,6 +11,17 @@ module Schedulable
       before_validation :force_schedule_start_in_config_timezone
     end
 
+    def schedule_multiloc
+      # We currently only support weekly schedules here.
+      weekly_day_key = "email_campaigns.schedules.weekly.#{schedule['rrules'][0]['validations']['day'][0]}"
+      weekly_hour = schedule['rrules'][0]['validations']['hour_of_day'][0]
+
+      multiloc = MultilocService.new.i18n_to_multiloc(weekly_day_key)
+      multiloc.each_value { |v| v << " @ #{weekly_hour}:00" }
```

Comment:
@kogre This works, but I really want to use interpolation, similar to how we interpolate values in the multilocs used in the mail views.

I tried using this value in the translation YAML:
```YAML
"Weekly on Wednesdays %{atHour}"
```
And then using this in the method here:
```Ruby
MultilocService.new.i18n_to_multiloc(weekly_day_key, atHour: " @ #{weekly_hour}:00")
```
[Docs](https://guides.rubyonrails.org/v6.1/i18n.html#:~:text=3.3%20Passing%20Variables%20to%20Translations) seem to suggest something like this is possible.

What am I doing wrong?

Reply:
Your are passing the argument to our homemade `MulticlocService`, not to `I18n.t` :upside_down_face:

Reply:
Thanks. Works with `I18n.t` indeed!

Reply:
> You're assuming that hour encoding is the same in all languages

I've addressed this though the use of specifying the format for each locale in the appropriate YAML files.

Unfortunately I18n doesn't give us a useful default method for getting a localised time string (and it also isn't great at getting English 12-hour clock formats correct or applying them to the correct locales).